### DISCUSSION TITLE
Add another parameter as filename like file path

### DISCUSSION THREAD

**Discussion by VulchiSarath** [boringtools/git-alerts#27] [Ideas]
Can you please add another parameter as file_name as file_path because right now we got static file names. If I want to give the different file name then we don't have an option for this. If I'm executing the same org names twice then the file will be override. So can you please add this new parameter?

**Comment by c0d3G33k** [comment]
Thanks for raising this @VulchiSarath, let me check this in sometime.

**Comment by c0d3G33k** [comment]
Hi @VulchiSarath,

`GitAlerts` generates multiple files during a scan. To support this request, I need to add two different parameters. I believe for the same organization, you can provide a different report path to prevent existing files from being overridden. Let me know your thoughts on this.
